fenix/app/src/main/java/org/mozilla/fenix/compose/tabstray/TabGridItem.kt

```diff
@@ -135,7 +137,7 @@ fun TabGridItem(
                         isContentRtl = BidiFormatter.getInstance().isRtl(tab.content.title),
                     ) {
                         Text(
-                            text = tab.content.title,
+                            text = tab.toDisplayTitle().take(MAX_URI_LENGTH),
```

Comment:
Is there ever a situation where we using the full length title, or are we always showing the display title? JW if it might be worth creating a `TabText` or something.

Reply:
From doing a quick search the title is used as the full length in: collections storage, navigation directions arguments, share sheet, suggestions, and custom tabs. There is likely more though.

I, too, had the thought of whether it made sense to create a short hand parameter for this, or cutting it down at the data layer.